packages/manager/apps/dedicated/client/app/license/translations/Messages_fr_FR.json

```diff
@@ -324,6 +324,16 @@
   "license_designation_WINDOWS_version_WINDOWS_SERVER_2016_DATACENTER_EDITION_1_CPU_10_CORES": "Microsoft Windows Server 2016 Datacenter Edition (1 CPU 10 CORES)",
   "license_designation_WINDOWS_version_windows-server-2016-license-datacenter-edition-1-cpu-10-cores": "Microsoft Windows Server 2016 Datacenter Edition (1 CPU 10 CORES)",
   "license_designation_WINDOWS_version_WINDOWS_SERVER_2016_DATACENTER_EDITION_1_CPU_12_CORES": "Microsoft Windows Server 2016 Datacenter Edition (1 CPU 12 CORES)",
+  "license_designation_WINDOWS_version_WINDOWS_SERVER_2016_DATACENTER_EDITION_1_CPU_14_CORES": "MICROSOFT Windows Server 2016 Datacenter edition (1 CPU - 14 CORES)",
+  "license_designation_WINDOWS_version_WINDOWS_SERVER_2016_DATACENTER_EDITION_1_CPU_16_CORES": "MICROSOFT Windows Server 2016 Datacenter edition (1 CPU - 16 CORES)",
+  "license_designation_WINDOWS_version_WINDOWS_SERVER_2016_DATACENTER_EDITION_1_CPU_18_CORES": "MICROSOFT Windows Server 2016 Datacenter edition (1 CPU - 18 CORES)",
+  "license_designation_WINDOWS_version_WINDOWS_SERVER_2016_DATACENTER_EDITION_1_CPU_20_CORES": "MICROSOFT Windows Server 2016 Datacenter edition (1 CPU - 20 CORES)",
+  "license_designation_WINDOWS_version_WINDOWS_SERVER_2016_DATACENTER_EDITION_1_CPU_22_CORES": "MICROSOFT Windows Server 2016 Datacenter edition (1 CPU - 22 CORES)",
+  "license_designation_WINDOWS_version_WINDOWS_SERVER_2016_DATACENTER_EDITION_1_CPU_24_CORES": "MICROSOFT Windows Server 2016 Datacenter edition (1 CPU - 24 CORES)",
+  "license_designation_WINDOWS_version_WINDOWS_SERVER_2016_DATACENTER_EDITION_1_CPU_32_CORES": "MICROSOFT Windows Server 2016 Datacenter edition (1 CPU - 32 CORES)",
+  "license_designation_WINDOWS_version_WINDOWS_SERVER_2016_DATACENTER_EDITION_1_CPU_48_CORES": "MICROSOFT Windows Server 2016 Datacenter edition (1 CPU - 48 CORES)",
+  "license_designation_WINDOWS_version_WINDOWS_SERVER_2016_DATACENTER_EDITION_1_CPU_6_CORES": "MICROSOFT Windows Server 2016 Datacenter edition (1 CPU - 6 CORES)",
+  "license_designation_WINDOWS_version_WINDOWS_SERVER_2016_DATACENTER_EDITION_1_CPU_64_CORES": "MICROSOFT Windows Server 2016 Datacenter edition (1 CPU - 64 CORES)",
```

Comment:
We must have `Microsoft` not `MICROSOFT`.
To update in the rest of Translations.

Reply:
Also, `core` is word not an acronym, so we have to write it `cores` not `CORES`

Reply:
done